src/Client.php

```diff
@@ -20,6 +20,8 @@ class Client
     const THUMBNAIL_SIZE_L = 'w640h480';
     const THUMBNAIL_SIZE_XL = 'w1024h768';
 
+    const MAX_CHUNK_SIZE = 157286400;
```

Comment:
It's very difficult to know how many megabytes this number represents

Could you rewrite this as `1024 * 1024 * <amount of megabytes>`?